sound/soc/intel/common/soc-acpi-intel-tgl-match.c

```diff
@@ -14,11 +14,25 @@ static struct snd_soc_acpi_codecs tgl_codecs = {
 	.codecs = {"MX98357A"}
 };
 
+static const u64 rt711_0_adr[] = {
+	0x000010025D071100
+};
+
+static const struct snd_soc_acpi_link tgl_rvp[] = {
+	{
+		.mask = BIT(0),
+		.dev_num = ARRAY_SIZE(rt711_0_adr),
+		.adr = rt711_0_adr,
+	},
+	{}
+};
+
```

Comment:
we should add the cases where we have 1 or 2  rt1308 present on the link1?

Reply:
> we should add the cases where we have 1 or 2 rt1308 present on the link1?

We support sdw rt711 with i2s rt1308 only for now. The snd_soc_acpi_intel_tgl_sdw_machines[] looks odd to me. I didn't test it, but it seems not working. We enable rt711 only here, but rt1308 and rt715 are required by machine driver.

Reply:
I thought @RanderWang added support for TGL with 2 SoundWire devices?

Reply:
> I thought @RanderWang added support for TGL with 2 RT1308 SoundWire devices on the same link?

I talked to @RanderWang and yes, he added support for TGL with 2 RT1308 SoundWire devices on the same link. He added 2 RT1308 on sdw link 0, however, he didn't know if it is what we need. Do we need to support rt711 SDW mode for TGL? i.e. RT711 on sdw link 0 and two RT1308 on sdw link 1.

Reply:
I have done two solutions on TGL. one is using native TGL, rt711 on link0,  two 1308 on link1. Another is using 3-in-1 board,  two 1308s on link0 for we don't know how to rework TGL to support external codec on link1. And there is no usable ICL in shanghai. @plbossart  will you get a TGL RVP ?

Reply:
yes for TGL we should support RT711 on link0 and 2 RT1308 on link1, this is one of the asks.

Reply:
> yes for TGL we should support RT711 on link0 and 2 RT1308 on link1, this is one of the asks.

Got it. I will update tgl match.

Reply:
@RanderWang Could you double check is tgl_rvp[] correct? Thanks.